src/ucp/wireup/wireup.c

```diff
@@ -124,12 +125,12 @@ ucs_status_t ucp_wireup_msg_progress(uct_pending_req_t *self)
 
     packed_len = uct_ep_am_bcopy(ep->uct_eps[req->send.lane], UCP_AM_ID_WIREUP,
                                  ucp_wireup_msg_pack, req, am_flags);
-    if (packed_len < 0) {
-        if (packed_len != UCS_ERR_NO_RESOURCE) {
-            ucs_error("failed to send wireup: %s",
-                      ucs_status_string((ucs_status_t)packed_len));
-        }
-        return (ucs_status_t)packed_len;
+    UCP_AM_BCOPY_HANDLE_STATUS(0, (ucs_status_t)packed_len);
```

Comment:
pls add () to UCP_AM_BCOPY_HANDLE_STATUS parameters usage

Reply:
done